### ISSUE TITLE
[a11y]: helperText on ComboBox has insufficient color contrast when disabled

### ISSUE THREAD

**Issue by crhuff-ibm** [carbon-design-system/carbon#11165]
### Package

carbon-components-react

### Browser

Chrome

### Operating System

MacOS

### Package version

7.54.2

### React version

^v17.0.0

### Automated testing tool and ruleset

IBM Equal Access Accessibility Checker - Latest

### Description

The helper text when a ComboBox component is set to disabled should have high enough contrast to past wcag 

### WCAG 2.1 Violation

1.4.3

### CodeSandbox example

https://codesandbox.io/s/pensive-water-j13swm?file=/src/index.js

### Steps to reproduce

Disable combobox
Add helper text

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by dakahn** [comment]
Text or images of text that are part of an inactive user interface components have no contrast requirement as seen [here in Understanding Contrast Requirements](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)
